docker-traceability-api/pom.xml

```diff
@@ -132,6 +73,20 @@
       <groupId>com.github.docker-java</groupId>
       <artifactId>docker-java</artifactId>
       <version>${docker-java.version}</version>
+      <exclusions>
+        <exclusion>
+          <groupId>org.apache.httpcomponents</groupId>
+          <artifactId>httpclient</artifactId>
+        </exclusion>
+        <exclusion>
+          <groupId>org.apache.httpcomponents</groupId>
+          <artifactId>httpmime</artifactId>
+        </exclusion>
+        <exclusion>
+          <groupId>org.glassfish.hk2</groupId>
+          <artifactId>hk2-utils</artifactId>
+        </exclusion>
+      </exclusions>
```

Comment:
You need compare library version and library usage to understand whether it save to replacements switches. 
But keeping in mind that this plugin has totally wrong dependency coming from docker-java+jenkins you can just merge and nobody will complain even if you broke it :)